lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -40,7 +40,9 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: gettext('edX learners can see my:'),
+                title: interpolate_text(
+                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
```

Comment:
Here's a long platform name which seems a little clumsy but it is also probably fine, so maybe this isn't an issue:

![image](https://cloud.githubusercontent.com/assets/5985072/7371027/2bf47140-ed8c-11e4-867d-83b975049d33.png)

Regarding i18n, there is a general problem with translating partial sentences and then assuming that other words can be added to it.  See section 10.1.1 here: http://edx-developer-guide.readthedocs.org/en/latest/internationalization/i18n.html

Maybe in all languages it will be possible to work with this to make something that seems reasonable, but my gut says that it would be better to not assume that this is a sentence that includes the drop down selection. I imagine there are some languages where the selected text would need to be in the middle of the sentence to make sense. I don't know for sure,  so I was just raising it in case someone else had an opinion.